### PR TITLE
chore(holonix): add github issue template

### PR DESCRIPTION
**Pull request by jost-s**
### Summary
Add an issue template for bugs related to Holonix.

### TODO:
- [ ] CHANGELOG(s) updated with appropriate info
- [ ] Just before pressing the merge button, ensure new entries to CHANGELOG(s) are still under the _UNRELEASED_ heading

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/holonix_bug_report.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+**Describe the bug**
+A clear and concise description of what the bug is, which dependency it is to do with,
+how you encountered it, and the text of any error you received.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**System information:**
+ - OS: [e.g. iOS]
```
Comment (steveeJ) [inline .github/ISSUE_TEMPLATE/holonix_bug_report.md:18]:
```suggestion
 - OS: [e.g. macOS 13, Ubuntu Linux 22.04]
```

File: .github/ISSUE_TEMPLATE/holonix_bug_report.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+name: Holonix bug report
+about: Create a report to help us improve the Holonix dev shell
+title: "[BUG - HOLONIX]"
+labels: ''
```
Comment (steveeJ) [inline .github/ISSUE_TEMPLATE/holonix_bug_report.md:5]:
```suggestion
labels: ["bug", "nix"]
```

**Review by steveeJ — COMMENTED** [review]
thanks for creating this!

i wonder whether the "clear and concise" might be off-putting so some.
could you try softening that a bit?

**Comment by jost-s** [comment]
> i wonder whether the "clear and concise" might be off-putting so some. could you try softening that a bit? (if you agree, that is)

Mmh, what about "fluffy and long-winded" instead? =D
I'm fine with what you prefer.

File: .github/ISSUE_TEMPLATE/holonix_bug_report.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+name: Holonix bug report
+about: Create a report to help us improve the Holonix dev shell
+title: "[BUG - HOLONIX]"
+labels: ["bug", "nix"]
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is, which dependency it is to do with,
```
Comment (steveeJ) [inline .github/ISSUE_TEMPLATE/holonix_bug_report.md:11]:
```suggestion
A description of what the bug is, which dependency it is to do with,
```

File: .github/ISSUE_TEMPLATE/holonix_bug_report.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+how you encountered it, and the text of any error you received.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
```
Comment (steveeJ) [inline .github/ISSUE_TEMPLATE/holonix_bug_report.md:15]:
```suggestion
A description of what you expected to happen.
```

File: .github/ISSUE_TEMPLATE/holonix_bug_report.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+**System information:**
+ - OS: [e.g. macOS 13, Ubuntu Linux 22.04]
+ - Nix version: (`nix --version`)
+
+**Steps to reproduce**
+Detailed, specific steps which will reliably reproduce this bug.
```
Comment (steveeJ) [inline .github/ISSUE_TEMPLATE/holonix_bug_report.md:22]:
```suggestion
Steps which will reliably reproduce this bug.
```

**Review by steveeJ — COMMENTED** [review]
this suggests removing most of the adjectives. i trust that every reporter will put in the best effort they can make at the time of reporting.

**Pull request merged**